docs/blog/2019-09-26-announcing-gatsby-15m-series-a-funding-round/index.md

```diff
@@ -61,7 +61,7 @@ Lots more announcements to share in the coming weeks and months 🚀
 
 ## Growing our team
 
-Gatsby is a distributed team with members around the world. There’s now 35 Gatsby Inkteam members, and with this new investment, we’ll be hiring more people to work on open source, write docs and tutorials, build cloud products, and create next-generation site design & building tools.
+Gatsby is a distributed team with members around the world. There’s now 35 Gatsby Inc team members, and with this new investment, we’ll be hiring more people to work on open source, write docs and tutorials, build cloud products, and create next-generation site design & building tools.
```

Comment:
Inkteam is the correct term.

Reply:
oh i heard `Inkteam` the first time and google also gives no answer about it...

Reply:
changed

Reply:
it's mostly an internal thing